### ISSUE TITLE
Integration with Atea?

### ISSUE THREAD

**Issue by pkamenarsky** [nhmood/watson-ruby#125]
Hi, I am the developer of [Atea](https://github.com/pkamenarsky/atea). First of all, great work! I see Watson is similar to what I had in mind when I wrote Atea and I'm really glad to see more text based task managers see the light of day.

In the future, when we both have some time, we could see if we can come up with some kind of integration scheme between the two projects; some people may find it useful. Alternatively, if somebody feels like it, patch requests are always welcome :)

What do you think?

**Comment by nhmood** [comment]
Absolutely! Atea looks great and I think the two would be a great fit for each other. 
When I get a chance I'll take a look at both code bases and see try to familiarize myself with Atea and then we should definitely chat about integrating the two when available.
